sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,282 @@
+- Made `sycl::exception` `nothrow` copy constructible. [289e33da]
+- Marked `has_property` methods as `noexcept`. [417b5a29]
+- Improved `sycl::event::get_profiling_info` exception message when `event` is
+  default constructed. [2e86cd459]
+- Added a diagnostic (in form of `static_assert`) about kernel lambda size
+  mismatch between host and device. [d278c671] [ec179b7a] [f417a88e]
+- Updated `pipes` class to throw exceptions if used on host. [eab29696]
+- Updated ESIMD Emulator PI plugin to report support for `cl_khr_fp64`
+  extension. [398571a5]
+- Updated Level Zero plugin to prefer copy engine for memory read/write
+  operations. [65c3ea29]
+- Optimized some memory transfers. [92d35cd1]
+- Enabled event caching in Level Zero PI plugin. [a41b33c3]
+- Optimized some reductions for discrete GPUs. [c22a5d3f]
+- Improved performance of event synchronization on CUDA backend. [c4f326aa]
+- Added ability to use descendent devices of context members can now be used
+  within that context. Not supported with OpenCL backend yet. [a0c8c503] [78a483c7]
+- Added support for querying `atomic64` device capability with HIP backend. [cb190fc2]
+- Enabled FTZ operations for CUDA/PTX backend via
+  `-fcyda-flush-denormals-to-zero`. [e8e7ae83]
+- Improved error message about incorrect kernel argument types with CUDA backend. [2542e6a8]
+- Limited allowed argument types for `rol/ror` ESIMD functions to better
+  represent HW capabilities. [b05f256c]
+- Implemented `mem_advise` reset and managed memory checks for CUDA backend. [fe18839c]
+- Added concurrent memory check to `mem_advise` on CUDA backend. [33746d8d]
+- Enabled multiple HIP streams per SYCL queue. [e0c40a9f]
+- Implemented lazy mechanism of setting context for default-constructed events. [ed92c4ca]
+
+### SYCL Compiler
+
+- Increased max `_Bitint` size to 4096 for FPGA target. [db5f72a8] [3f06cad0]
+- Removed deprecation message for `[[intel::disable_loop_pipelining]]` attribute. [07201f56]
+- Allowed `__builtin_assume_aligned` to be called from device code. [24937eac]
+- Improved link step performance when `per_kernel` device code split is used. [84de9d6d]
+- Added support for `SYCL_EXTERNAL` on `device_global` variables. [8b958f67]
+- Updated `__builtin_intel_fpga_mem` to accept more parameters. [231338dc]
+- Updated `ivdep` attribute to allow `safelen = 0`. [558b3ba4]
+- Improved linking with `sycl.lib` on Windows. [404d2816]
+- Implemented more diagnostics about incorrect `device_global` usages. [12657218]
+- Improved library resolution for `libsycl.so`. [4ce19d69]
+- Improved diagnostics when linkind with mismatched objects. [0e0202ee]
+- Added a warning for floating-point size changes after implicit conversions. [e4f5d55f]
+- Made `invoke_simd` convert its argument to appropriate types. [038764fd]
+
+### Documentation
+
+- Removed explicit `cl` namespace references. [433ea5c7]
+- Added a short guildeline on using CMake with SYCL compiler. [fa603c3e]
```

Comment:
```suggestion
- Added a short guideline on using CMake with SYCL compiler. [fa603c3e]
```

Reply:
Applied in ed5e71d464608b7be8fd28ee14249979e67e4fb8